bff-tiltaksarrangor/src/main/kotlin/no/nav/amt/tiltak/bff/tiltaksarrangor/dto/DeltakerlisteDto.kt

```diff
@@ -1,16 +1,21 @@
 package no.nav.amt.tiltak.bff.tiltaksarrangor.dto
 
 import no.nav.amt.tiltak.core.domain.tiltak.Gjennomforing
+import java.time.LocalDate
 import java.util.UUID
 
 data class DeltakerlisteDto(
 	val id: UUID,
 	val navn: String,
-	val type: String
+	val type: String,
+	val startdato: LocalDate?,
+	val sluttdato: LocalDate?
 )
 
 fun Gjennomforing.toDeltakerlisteDto() = DeltakerlisteDto(
 	id = id,
 	navn = navn,
-	type = tiltak.navn
+	type = tiltak.navn,
+	startdato = if (erKurs) startDato else null,
```

Comment:
Kan ikke datoene settes selv om det ikke er kurs? Vil tro dette kan skape forvirring senere. Visning av datoer føler jeg er frontend oppgave

Reply:
jo, de kan det, men jeg tenkte det var digg for frontend å slippe denne logikken..?

Reply:
endrer :)